packages/@aws-cdk/aws-ecs/lib/base/task-definition.ts

```diff
@@ -208,6 +209,16 @@ export interface TaskDefinitionProps extends CommonTaskDefinitionProps {
    * @default - Undefined, in which case, the task will receive 20GiB ephemeral storage.
    */
   readonly ephemeralStorageGiB?: number;
+
+  /**
+   * The operating system that your task definitions are running on.
+   * A runtimePlatform is supported only for tasks using the Fargate launch type.
+   *
+   * When you specify a task in a service, this value must match the runtimePlatform value of the service.
```

Comment:
```suggestion
```

I think it's unclear in the cloudformation document and could be confusing as there is actually no `runtimePlatform` property for the service resource. Let's just remove it. @madeline-k  wdyt?